src/utils/isRangeContextMediaFeature.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Check whether a media feature is a range context one
+ *
+ * @param {string} media feature
+ * @return {boolean} If `true`, media feature is a range context one
+ */
+export default function (mediaFeature) {
+  return (
+    mediaFeature.indexOf("=") !== -1
+    || mediaFeature.indexOf("<") !== -1
+    || mediaFeature.indexOf(">") !== -1
```

Comment:
This works fine. If anybody wants to dig into micro-optimizations, we could do all this in one loop, or see if a regexp is faster.